enclave/src/openfinex/market.rs

```diff
@@ -0,0 +1,73 @@
+// This file is part of Polkadex.
+
+// Copyright (C) 2020-2021 Polkadex oü.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+use serde::{Deserialize, Serialize};
+use std::{string::String, string::ToString};
+
+//{"id":"btcusd","name":"BTC/USD","base_unit":"btc","quote_unit":"usd","state":"enabled","amount_precision":4,
+// "price_precision":4,"min_price":"0.0001","max_price":"0","min_amount":"0.0001","position":100,"filters":[]}
+
+/// Market object given by OpenFinex as JSON string
+#[derive(Serialize, Deserialize, Debug, PartialEq, Eq, Clone)]
+pub struct Market {
+    pub id: String,
+    pub name: String,
+    pub base_unit: String,
+    pub quote_unit: String,
+    pub state: MarketState,
+    pub amount_precision: u128,
+    pub price_precision: u128,
+    //min_price: String,
+    //max_price: String,
+    //min_amount: String,
+    //position: u128,
+}
```

Comment:
New Market struct to hold the information we get from OpenFinex. Only maps a part of the information we get back from OpenFinex, the rest we ignore for now since it's not relevant at this point